package-lock.json

```diff
@@ -1789,9 +1779,9 @@
       }
     },
     "lodash": {
-      "version": "4.17.11",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.11.tgz",
-      "integrity": "sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg=="
+      "version": "4.17.15",
+      "resolved": "https://mix.jfrog.io/mix/api/npm/npm-remote/lodash/-/lodash-4.17.15.tgz",
```

Comment:
is the change to `mix.jfrog.io` intentional?

Reply:
🤦‍♂